File: 프로그래머스/lv2/12924. 숫자의 표현/숫자의 표현.py
def solution(n):
    answer = 0
    for i in range(1,n+1):
        a=i
        while a<=n:
            if a==n:
                answer+=1
                break
            elif i<n:
                i+=1
                a+=i
            
            
        
                
    return answer

File: 프로그래머스/2/12924. 숫자의 표현/숫자의 표현.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        int start = 1;
        while(start<=n){
            int sum =0;
            for(int i=start;i<=n;i++){
                sum+=i;
                if(sum==n){
                    answer++;
                }else if(sum<n){
                    continue;
                }else{
                    break;
                }
            }
            start++;
        }
        return answer;
    }
}